.github/workflows/build_tester_base_image.yml

```diff
@@ -0,0 +1,41 @@
+name: build-tester-base-image
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # trigger at 1:30 am every day
+    - cron:  '30 1 * * *'
+
+concurrency:
+  group: build-tester-base-image
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  build-docker:
+    runs-on: ubuntu-latest
+    if: github.repository == 'geodynamics/aspect'
+    steps:    
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_LOGIN }}
+
+      - name: Build and push Docker image for main
+        uses: docker/build-push-action@v3
+        with:
+          context: ./contrib/docker/tester/
+          cache-from: type=registry,ref=ubuntu:20.04
+          cache-to: type=inline
+          push: true
+          tags: geodynamics/aspect-tester:latest
```

Comment:
Should we add deal.II or "master" or something to the name? It shows up in the description of the runs on each pull request.